bower.json

```diff
@@ -22,7 +22,8 @@
   "license": "https://github.com/cfpb/cf-typography/blob/gh-pages/TERMS.md",
   "main": "src/cf-typography.less",
   "dependencies": {
-    "cf-colors": "git://github.com/cfpb/cf-colors.git",
+    "cf-core": "latest",
+    "cf-icons": "latest",
     "normalize-css": "~3.0.0",
```

Comment:
For consistency with the other components can you ditch the two normalize deps here please?